common/game_rules/aroai_game_rules.txt

```diff
@@ -0,0 +1,129 @@
+﻿aroai_power_level_of_ai_countries = {
+    default = aroai_power_level_of_ai_countries_100
+    aroai_power_level_of_ai_countries_0 = {}
+    aroai_power_level_of_ai_countries_10 = {}
+    aroai_power_level_of_ai_countries_20 = {}
+    aroai_power_level_of_ai_countries_30 = {}
+    aroai_power_level_of_ai_countries_40 = {}
+    aroai_power_level_of_ai_countries_50 = {}
+    aroai_power_level_of_ai_countries_60 = {}
+    aroai_power_level_of_ai_countries_70 = {}
+    aroai_power_level_of_ai_countries_80 = {}
+    aroai_power_level_of_ai_countries_90 = {}
+    aroai_power_level_of_ai_countries_100 = {}
+}
+
+aroai_power_level_of_player_subjects = {
+    default = aroai_power_level_of_player_subjects_same
+    aroai_power_level_of_player_subjects_0 = {}
+    aroai_power_level_of_player_subjects_10 = {}
+    aroai_power_level_of_player_subjects_20 = {}
+    aroai_power_level_of_player_subjects_30 = {}
+    aroai_power_level_of_player_subjects_40 = {}
+    aroai_power_level_of_player_subjects_50 = {}
+    aroai_power_level_of_player_subjects_60 = {}
+    aroai_power_level_of_player_subjects_70 = {}
+    aroai_power_level_of_player_subjects_80 = {}
+    aroai_power_level_of_player_subjects_90 = {}
+    aroai_power_level_of_player_subjects_100 = {}
+    aroai_power_level_of_player_subjects_same = {}
+}
+
+aroai_power_level_of_ai_china = {
+    default = aroai_power_level_of_ai_china_100
+    aroai_power_level_of_ai_china_0 = {}
+    aroai_power_level_of_ai_china_10 = {}
+    aroai_power_level_of_ai_china_20 = {}
+    aroai_power_level_of_ai_china_30 = {}
+    aroai_power_level_of_ai_china_40 = {}
+    aroai_power_level_of_ai_china_50 = {}
+    aroai_power_level_of_ai_china_60 = {}
+    aroai_power_level_of_ai_china_70 = {}
+    aroai_power_level_of_ai_china_80 = {}
+    aroai_power_level_of_ai_china_90 = {}
+    aroai_power_level_of_ai_china_100 = {}
```

Comment:
This game rule was added because players were complaining they couldn't colonize india and china hard enough.  If anything, I'd say we remove it entirely.